warp-academy-docs/standards/atomic-asset-impl.md

```diff
@@ -0,0 +1,90 @@
+# Introduction
```

Comment:
is there any info here about adding proper tags? shouldn't we include it in the specs so it is retrievable via aggregate node?

Reply:
Goooood catch, added!